components/partials/Header.js

```diff
@@ -0,0 +1,176 @@
+            <input
+              className="!border-0 text-sm px-5 !bg-slate-100 !w-52"
+              placeholder="Search"
+              type="text"
+              // value=""
+            />
+          </div>
+
+          <div className="flex grow items-center">
+            <ul className="flex flex-row flex-wrap items-center justify-start my-0 p-0 mb-3 sm:my-2 lg:ml-auto text-base lg:text-[1.125rem]">
+              <li className="my-2 mr-4 font-normal">
+                <Link
+                  href="/"
+                  className={
+                    router.pathname == "/"
+                      ? `md:mr-6 xl:mr-9 relative pb-[5.5px] text-transparent bg-clip-text bg-gradient-to-r after:bg-gradient-to-r from-pink-300 to-orange-300 after:absolute after:bottom-0 after:left-0 after:w-full after:h-[2px]`
+                      : `md:mr-6 xl:mr-9 text-black-900 hover:relative pb-[5.5px] hover:text-transparent hover:bg-clip-text hover:bg-gradient-to-r hover:after:bg-gradient-to-r from-pink-300 to-orange-300 hover:after:absolute hover:after:bottom-0 hover:after:left-0 hover:after:w-full hover:after:h-[2px]`
+                  }
+                >
+                  Featured
+                </Link>
+              </li>
+
+              <li className="my-2 mr-4 font-normal ">
+                <Link
+                  href="/subscribe"
+                  className={
+                    router.pathname == "/subscribe"
+                      ? "md:mr-6 xl:mr-9 relative pb-[5.5px] text-transparent bg-clip-text bg-gradient-to-r after:bg-gradient-to-r from-pink-300 to-orange-300 after:absolute after:bottom-0 after:left-0 after:w-full after:h-[2px]"
+                      : "md:mr-6 xl:mr-9 text-black-900 hover:relative pb-[5.5px] hover:text-transparent hover:bg-clip-text  hover:bg-gradient-to-r hover:after:bg-gradient-to-r from-pink-300 to-orange-300 hover:after:absolute hover:after:bottom-0 hover:after:left-0 hover:after:w-full hover:after:h-[2px]"
+                  }
+                >
+                  Subscribe
+                </Link>
+              </li>
+
+              <li className="my-2 mr-4 font-normal ">
+                <Link
+                  href="/write"
+                  className={
+                    router.pathname == "/write"
+                      ? "md:mr-6 xl:mr-9 relative pb-[5.5px] text-transparent bg-clip-text bg-gradient-to-r after:bg-gradient-to-r from-pink-300 to-orange-300 after:absolute after:bottom-0 after:left-0 after:w-full after:h-[2px]"
+                      : "md:mr-6 xl:mr-9 text-black-900 hover:relative pb-[5.5px] hover:text-transparent hover:bg-clip-text  hover:bg-gradient-to-r hover:after:bg-gradient-to-r from-pink-300 to-orange-300 hover:after:absolute hover:after:bottom-0 hover:after:left-0 hover:after:w-full hover:after:h-[2px]"
+                  }
+                >
+                  Write
+                </Link>
+              </li>
+
+              {/* ==== If User Not Login ====  */}
```

Comment:
Remove commented code.

Reply:
Actually this code is important because when we will create api we will use this code e.g. **If** User Login (Display **Login**) **Else** (Display **Profile**)